querybook/webapp/components/ListDataDoc/components/LastRun.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import moment from 'moment';
+
+export const LastRun: React.FunctionComponent<{ createdAt: number }> = ({
+    createdAt,
+}) => {
+    return createdAt ? (
+        <div
+            title={`${moment
```

Comment:
use generateFormattedDate and fromNow

Reply:
@czgu The generateFormattedDate returns local time but we decided to use UTC. Do you think, a better option is Local time?